tests/kernel/mem_protect/sys_sem/prj.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_SMP=n
+CONFIG_QEMU_TICKLESS_WORKAROUND=y
```

Comment:
Should this be upstreamed?

Reply:
no, I was just testing this, but it didn't solve the sanitycheck errors.    see conversation here: https://teams.microsoft.com/l/message/19:e7aa575b8c2a4438aecc6daa5b380ef8@thread.skype/1570002786674?tenantId=28e5afa2-bf6f-419a-8cf6-b31c6e9e5e8d&groupId=bbf3a6ae-b458-4b7b-9c01-2d2881d8b2fb&parentMessageId=1570002786674&teamName=NCS&channelName=General&createdTime=1570002786674